ChameleonForms.Example/Controllers/ExampleFormsController.cs

```diff
@@ -209,25 +229,32 @@ public ViewModelExample()
         public bool Boolean { get; set; }
 
         public ChildViewModel Child { get; set; }
+
+        [DisplayFormat(DataFormatString = "{0:g}", ApplyFormatInEditMode = true)]
+        public DateTime DateField { get; set; }

```

Comment:
I wonder if we should name this something more descriptive to make it obvious what's being tested when using this property e.g. `DateFieldWithShortDateFormatString` or something like that?